test/lib/cli/commands/config.js

```diff
@@ -0,0 +1,120 @@
+import test from "ava";
+import sinon from "sinon";
+import esmock from "esmock";
+const {default: fs} = await import("graceful-fs");
```

Comment:
I think we can use the native "fs" for this test and remove the graceful-fs dependency from package.json again

Reply:
Not needed anymore at all as we now use entirely the Configuration class from the ui5-project